src/wp-includes/class-walker-nav-menu.php

```diff
@@ -73,9 +73,27 @@ public function start_lvl( &$output, $depth = 0, $args = null ) {
 		 * @param int      $depth   Depth of menu item. Used for padding.
 		 */
 		$class_names = implode( ' ', apply_filters( 'nav_menu_submenu_css_class', $classes, $args, $depth ) );
-		$class_names = $class_names ? ' class="' . esc_attr( $class_names ) . '"' : '';
 
-		$output .= "{$n}{$indent}<ul$class_names>{$n}";
+		$atts 		   = array();
+		$atts['class'] = ! empty( $class_names ) ? $class_names : '';
+
+		/**
+		 * Filters the HTML attributes applied to a menu item's ul element.
+		 *
+		 * @since 6.1.2
+		 *
+		 * @param array $atts {
+		 *     The HTML attributes applied to the `<ul>` element, empty strings are ignored.
+		 *
+		 *     @type string $class    HTML CSS class attribute
+		 * }
+		 * @param stdClass $args      An object of `wp_nav_menu()` arguments.
+		 * @param int      $depth     Depth of menu item. Used for padding.
+		 */
+		$atts 		= apply_filters( 'nav_menu_submenu_attributes', $atts, $args, $depth );
+		$attributes = $this->build_atts($atts);
```

Comment:
```suggestion
		$attributes = $this->build_atts( $atts );
```
Should fix these PHPCS failures.